src/utilities/helpers.ts

```diff
@@ -10,3 +10,10 @@ export const getThumbnailUrl = (url: string): string => {
 
   return url;
 };
+
+export const formatMessage = (message: string): string => {
```

Comment:
I think we might want a utility package that does this work for us. Urls can be much more complex. Here are two that might be worth checking on: https://www.npmjs.com/package/react-anchorme or http://markdown-it.github.io/linkify-it/doc/